episodes/contact-matrices.Rmd

```diff
@@ -0,0 +1,228 @@
+---
+title: 'Contact matrices'
+teaching: 10
+exercises: 2
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- What is a contact matrix?
+- How are contact matrices estimated?
+- How are contract matrices used?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+- Use the R package `socialmixr` to estimate a contact matrix
+- Understand the different types of analysis contact matrices can be used for 
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+<!-- ::::::::::::::::::::::::::::::::::::: prereq -->
+
+<!-- ## Prerequisites -->
+
+
+
+<!-- ::::::::::::::::::::::::::::::::: -->
+
+
+## Introduction
+
+Heterogeneity of contact patterns between different groups affects disease transmission. The rate of contact within and between groups can be presented in a contact matrix. In this tutorial we are going to learn how contact matrices can be used in different analyses and how the `{socialmixr}` package can be used to estimate contact matrices. 
+
+
+```{r,message=FALSE,warning=FALSE}
+library(socialmixr)
+```
+
+
+:::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::: instructor
+
+
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+## The contact matrix
+
+A contact matrix represents the amount of contact or mixing within and between subgroups. The subgroups are often age categories but can also be geographic areas or high/low risk groups. For example, a contact matrix representing the number of contacts per day in children and adults could be:
+
+$$
+\begin{bmatrix}
+8 & 2\\
+2 & 7 
+\end{bmatrix}
+$$
+Children contact 8 other children each day, adults contact 7 other adults and children contact 2 adults (and vice versa). We can use this kind of information to account for the role heterogeneity in contact plays in infectious disease transmission. 
+
+
+## Estimating contact matrices
+
+Contact matrices can be estimated from surveys or contact data, or synthetic ones can be used. The POLYMOD survey measured contact patterns in 8 European countries using data on the location and duration of contacts reported by the study participants [(Mossong et al. 2008)](https://doi.org/10.1371/journal.pmed.0050074). [Prem et al. 2021](https://doi.org/10.1371/journal.pcbi.1009098) used the POLYMOD data within a Bayesian hierarchical model to project contact matrices for 177 other countries. 
```

Comment:
I would maybe leave the different methods for generating contact matrices for later and then describe in more detail (and probably at some stage introduce the contactmatrix package, once we have a common contact matrix class, which is work in progress).

```suggestion
Contact matrices can be estimated from diary data. For example, the POLYMOD survey measured contact patterns in 8 European countries using data on the location and duration of contacts reported by the study participants [(Mossong et al. 2008)](https://doi.org/10.1371/journal.pmed.0050074).
```